controllers/movieController.js

```diff
@@ -0,0 +1,59 @@
+
+const movieService = require('../services/movieService');
+
+const errorhandler = (err,res) => {
+    return res.status(err.statusCode || 500).json({ message : err.message });
+};
+
+const getDetail = async (req, res) => {
+    try {
+        const { movieId } = req.params
+        
+        const listByMovieId = await movieService.getMovieDetail(movieId);
```

Comment:
```suggestion
        const movie = await movieService.getMovieDetail(movieId);
```
애초에 하나의 movie만 가져오는데 list라는 네이밍은 맞지 않는 것 같습니다.

Reply:
네 코멘트 감사합니다